src/register/RegistrationFields/CountryField/CountryField.test.jsx

```diff
@@ -172,7 +172,7 @@ describe('CountryField', () => {
       const dropdownItem = container.querySelector('.dropdown-item');
       fireEvent.click(dropdownItem);
 
-      expect(props.onChangeHandler).toHaveBeenCalledTimes(1);
+      expect(props.onChangeHandler).toHaveBeenCalledTimes(2);
```

Comment:
Why is it called two times?

Reply:
I have changed the logic now when country field component is mounted then this check```!selectedCountry.countryCode``` gets true and call the ```onChangeHandler``` with blank country code and then we trigger click event which set the country field value. so In short ```onChangeHandler``` is called twice one for default setup value and second for when we set the value. I have removed the logic from ```ConfigurableRegistrationForm ``` and put it in ```CountrField``` component